website/docs/r/cloudant.html.markdown

```diff
@@ -0,0 +1,131 @@
+* `guid` - (String) Guid of resource instance.
+* `id` - (Optional, String) The unique identifier of the new Cloudant resource.
+* `last_operation` - (Map) The status of the last operation requested on the instance.
+* `locked` - (Boolean) A boolean that dictates if the resource instance should be deleted (cleaned up) during the processing of a region instance delete call.
+* `plan_history` - (List of Object) The plan history of the instance.
+* `resource_aliases_url` - (String) The relative path to the resource aliases for the instance.
+* `resource_bindings_url` - (String) The relative path to the resource bindings for the instance.
+* `resource_controller_url` - (String) The URL of the IBM Cloud dashboard that can be used to explore and view details about the resource.
+* `resource_crn` - (String) The crn of the resource.
+* `resource_group_crn` - (String) The long ID (full CRN) of the resource group.
+* `resource_group_name` - (String) The resource group name in which resource is provisioned.
+* `resource_id` - (String) The unique ID of the offering.
+* `resource_keys_url` - (String) The relative path to the resource keys for the instance.
+* `resource_name` - (String) The name of the resource.
+* `resource_plan_id` - (String) The unique ID of the plan associated with the offering.
+* `resource_status` - (String) The status of the resource.
+* `restored_at` - (String) The date when the instance under reclamation was restored.
+* `restored_by` - (String) The subject who restored the instance back from reclamation.
+* `scheduled_reclaim_at` - (String) The date when the instance was scheduled for reclamation.
+* `scheduled_reclaim_by` - (String) The subject who initiated the instance reclamation.
+* `service` - (String) The service type of the instance.
+* `state` - (String) The current state of the instance.
+* `status` - (String) Status of resource instance.
+* `sub_type` - (String) The sub-type of instance, e.g. cfaas .
+* `target_crn` - (String) The full deployment CRN as defined in the global catalog.
+* `throughput` - (Map of Number) Schema for detailed information about throughput capacity with breakdown by specific throughput requests classes.
+* `type` - (String) The type of the instance, e.g. service_instance.
+* `update_at` - (String) The date when the instance was last updated.
+* `update_by` - (String) The subject who updated the instance.
+
+## Import
+
+You can import the `ibm_cloudant` resource by using `crn`. An array of event types that are being sent to IBM Cloud Activity Tracker for the IBM Cloudant instance. Allowable values: management, data. "management" is a required element of this array.
+
+### Syntax
+
+```hcl-terraform
+$ terraform import ibm_cloudant.mycloudant <crn>
+```
+
+### Example
+```hcl
+$ terraform import ibm_cloudant.mycloudant "crn:v1:bluemix:public:cloudantnosqldb:us-south:a/4ea2849a2d3401ed1e457268541966ea:79262bd4-4067-4783-b5ce-b1dba84ff8c4::"
```

Comment:
Perhaps we should use something less real
```suggestion
$ terraform import ibm_cloudant.mycloudant "crn:v1:bluemix:public:cloudantnosqldb:us-south:a/abc123abc123abc123abc1:abc123ab-1234-1234-abc1-abc123abc123::"
```

Reply:
Changed in 0eafc34f